src/Router.ts

```diff
@@ -1,15 +1,12 @@
-import {
-  IGlobalConfiguration,
-  IMountableItem,
-  ICacheEngine,
-  IConstructorRouteOptions,
-} from '.';
-import Route from './Route';
-import express from 'express';
-import bodyParser from 'body-parser';
+import * as express from 'express';
+import * as bodyParser from 'body-parser';
 import axios, { AxiosInstance, AxiosRequestConfig } from 'axios';
 import { parse, DocumentNode, getOperationAST } from 'graphql';
-import { version } from './package.json';
+
+import Route from './Route';
+import { IGlobalConfiguration, IMountableItem, IConstructorRouteOptions } from './types';
+
+const version = require('../package.json').version;
```

Comment:
Should this be import?

Reply:
Yes but added a comment to fix in future pr since its further modifying ts configs